pom.xml

```diff
@@ -25,8 +25,15 @@
 		<sonar.jacoco.reportPath>../target/jacoco.exec</sonar.jacoco.reportPath>
 		<sonar.jacoco.itReportPath>../../../target/jacoco-it.exec</sonar.jacoco.itReportPath>
 		<sonar.jacoco.reportPaths>${sonar.jacoco.reportPath},${sonar.jacoco.itReportPath}</sonar.jacoco.reportPaths>
+		<!-- by default, don't run the deploy-to-jboss.org profile for all poms; set this to false in *updatesite/pom.xml -->
+		<maven.deploy.skip>true</maven.deploy.skip>
+		<skipDeployToJBossOrg>true</skipDeployToJBossOrg>
+		<skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
+		<filemgmtJbossOrg>10.5.105.197</filemgmtJbossOrg>
+		<jbossTychoPluginsVersion>1.1.0</jbossTychoPluginsVersion>
+		<jbosstoolsRelengPublishVersion>4.6.0-SNAPSHOT</jbosstoolsRelengPublishVersion>
```

Comment:
I think we should avoid to use snapshot of Jboss in this project to avoid strong dependency

Reply:
4.6.0 will be out soon as we get ready for .Final at the end of the current sprint.